src/web/releases.rs

```diff
@@ -273,75 +273,61 @@ fn get_search_results(
-                crates.github_stars,
-                SUM(releases.downloads) AS downloads,
-                -- Get the total number of results, disregarding the limit
-                COUNT(*) OVER() as total,
-
-                -- The levenshtein distance between the search query and the crate's name
-                levenshtein_less_equal(CAST($1 AS TEXT), CAST(crates.name AS TEXT), 3) as distance,
-                -- The similarity of the tokens of the search vs the tokens of `crates.content`.
-                -- The `32` normalizes the number by using `rank / (rank + 1)`
-                ts_rank_cd(crates.content, plainto_tsquery($1), 32)  as content_rank
-            FROM
-                crates
-                INNER JOIN releases ON releases.crate_id = crates.id
-                INNER JOIN (
-                    SELECT DISTINCT ON (crate_id)
-                        crate_id,
-                        version,
-                        description,
-                        target_name,
-                        release_time,
-                        rustdoc_status,
-                        yanked
-                    FROM
-                        releases
-                    ORDER BY
-                        crate_id,
-                        release_time DESC
-                ) AS latest_release ON latest_release.crate_id = crates.id
-
-            -- Filter crates that haven't been built and crates that have been yanked and
-            -- crates that don't match the query closely enough
-            WHERE
-                latest_release.rustdoc_status
-                AND NOT latest_release.yanked
-                AND (
-                    -- Crates names that match the query sandwiched between wildcards will pass
-                    crates.name ILIKE CONCAT('%', $1, '%')
-                    -- Crate names with which the levenshtein distance is closer or equal to 3 will pass
-                    OR levenshtein_less_equal($1, crates.name, 3) <= 3
-                    -- Crates where their content matches the query will pass
-                    OR plainto_tsquery($1) @@ crates.content
-                )
-            
-            GROUP BY crates.id, releases.id
-            
-            -- Ordering is prioritized by how closely the query matches the name, how closely the
-            -- query matches the description, and finally how many downloads the crate has
-            -- NOTE: this means that exact matches will be shown first
-            ORDER BY distance DESC,
-                content_rank DESC,
-                downloads_total DESC
-            
-            -- Allows pagination
-            LIMIT $2 OFFSET $3";
+    let statement = "
+        SELECT
+            crates.name,
+            releases.version,
+            releases.description,
+            releases.target_name,
+            releases.release_time,
+            releases.rustdoc_status,
+            SUM(crates.github_stars),
```

Comment:
This doesn't look quite right, we only want the latest stars ... All the entries should be the same anyway since I don't think the stars are linked to a specific release.